README.md

```diff
@@ -1,67 +1,69 @@
-# Proposed members' charter
+# Proposed students' charter
 
 Founders & Coders C.I.C. (FAC)  is a not-for-profit company whose main purpose is to provide study and training programmes for the general public and to provide workspace for its members.
 
-FAC was founded on the principal of reciprocity. That is, the company does not charge fees for its study programmes, but expects its members to contribute according to their abilities to the viability and development of the company.
+FAC was founded on the principal of reciprocity. That is, the company does not charge fees for its study programmes, but expects its students to contribute to the viability and development of the company in return.
 
-In particular, members are expected to participate in our study programmes and to support our goal of being financially sustainable, which will ensure that we can continue to offer study programmes and workspace free of charge long into the future.
+In particular, members of each cohort of students are expected to take the lead on facilitating the course for the next cohort, which will ensure that we can continue to offer study programmes and workspace free of charge long into the future. Our goal is to be financialy sustainable, and the company cannot survive without the continued support of our graduates.
```

Comment:
* 'graduates' / 'members of each cohort' are synonymous here.  Perhaps choose one for clarity.
* Not clear what is meant by 'facilitating the course' - does this refer specifically to remaining in the space and teaching, or should this be interpreted more loosely (e.g. contributing to resources)?  If the latter, this should be explicit, not implied.